2020-05-19 20:06:12,290 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:51]
2020-05-19 20:10:12,305 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:51]
2020-05-19 20:10:59,034 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:51]
2020-05-19 20:11:37,500 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:51]
2020-05-19 20:12:32,729 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:51]
2020-05-19 20:13:46,818 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:51]
2020-05-19 20:19:04,531 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:51]
2020-05-19 20:19:35,685 ERROR: Exception on /reset_password [GET] [in D:\learning\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\learning\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
TypeError: reset_password() missing 1 required positional argument: 'token'
2020-05-19 20:20:43,626 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:51]
2020-05-19 20:20:47,679 ERROR: Exception on /reset_password [GET] [in D:\learning\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\learning\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
TypeError: reset_password() missing 1 required positional argument: 'token'
2020-05-19 20:21:19,272 ERROR: Exception on /reset_password [GET] [in D:\learning\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\learning\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
TypeError: reset_password() missing 1 required positional argument: 'token'
2020-05-19 20:22:02,654 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:51]
2020-05-19 20:25:29,440 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:51]
2020-05-19 20:29:59,983 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:51]
2020-05-19 20:30:05,699 ERROR: Exception on /index [GET] [in D:\learning\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\learning\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\learning\microblog\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "D:\learning\microblog\app\routes.py", line 29, in index
    return render_template('index.html', title='Home', form=form,
  File "D:\learning\microblog\venv\lib\site-packages\flask\templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "D:\learning\microblog\venv\lib\site-packages\jinja2\environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\learning\microblog\venv\lib\site-packages\jinja2\environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "D:\learning\microblog\venv\lib\site-packages\jinja2\environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "D:\learning\microblog\venv\lib\site-packages\jinja2\loaders.py", line 127, in load
    code = environment.compile(source, name, filename)
  File "D:\learning\microblog\venv\lib\site-packages\jinja2\environment.py", line 638, in compile
    self.handle_exception(source=source_hint)
  File "D:\learning\microblog\venv\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "D:\learning\microblog\venv\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "D:\learning\microblog\app\templates\index.html", line 29, in template
    {% endblock %}
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'endblock'. You probably made a nesting mistake. Jinja is expecting this tag, but currently looking for 'elif' or 'else' or 'endif'. The innermost block that needs to be closed is 'if'.
2020-05-19 20:30:11,664 ERROR: Exception on /index [GET] [in D:\learning\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\learning\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\learning\microblog\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "D:\learning\microblog\app\routes.py", line 29, in index
    return render_template('index.html', title='Home', form=form,
  File "D:\learning\microblog\venv\lib\site-packages\flask\templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "D:\learning\microblog\venv\lib\site-packages\jinja2\environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\learning\microblog\venv\lib\site-packages\jinja2\environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "D:\learning\microblog\venv\lib\site-packages\jinja2\environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "D:\learning\microblog\venv\lib\site-packages\jinja2\loaders.py", line 127, in load
    code = environment.compile(source, name, filename)
  File "D:\learning\microblog\venv\lib\site-packages\jinja2\environment.py", line 638, in compile
    self.handle_exception(source=source_hint)
  File "D:\learning\microblog\venv\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "D:\learning\microblog\venv\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "D:\learning\microblog\app\templates\index.html", line 29, in template
    {% endblock %}
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'endblock'. You probably made a nesting mistake. Jinja is expecting this tag, but currently looking for 'elif' or 'else' or 'endif'. The innermost block that needs to be closed is 'if'.
2020-05-19 20:31:53,923 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:51]
